sources/gov/kaunas/municipality/krepsinio_aiksteles.yml

```
@@ -0,0 +1,8 @@
+---
+id: "gov/kaunas/municipality/krepsinio_aiksteles"
+title: "Krepšinio aikštelės"
+type: "source"
+source: "csv+https://raw.githubusercontent.com/Kauno-miestas/Laisvalaikio-zonos/master/Krepsinio%20aiksteles%20(2016).csv"
+provider: "gov/kaunas/municipality"
+since: "2018-10-06"
+stars: 2
```

Why two stars if this is a csv file?
```suggestion
stars: 3
```